cwf/gateway/helm/cwf-kubevirt/README.md

```diff
@@ -78,7 +78,7 @@ This chart installs the Magma CWF Gateway.
 
```

Comment:
Line 12 of this file: `docker_registry: docker.io/cwf_`. Should the "_" be replaced with a "-"? The same prefix is also still in helm/cwf-kubevirt/values.yaml and  helm/cwf-virtlet/README.md.

Reply:
The uploaded packages keep their names, so this is correct.